roles/ansible/tower/manage-workflow-templates/README.md

```diff
@@ -0,0 +1,124 @@
+manage-workflow-templates
+====================
+
+This role manages Workflow Templates for Ansible Tower, to be used with Ansible Tower runs.
+
+## Requirements
+
+A running Ansible Tower with admin permission level access.
+
+
+## Role Variables
+
+The variables used must be defined in the Ansible Inventory using the `ansible_tower.workflow_templates` list as explained below.
+
+| Variable | Description | Required | Defaults |
+|:---------|:------------|:---------|:---------|
+|ansible_tower.admin_password|Admin password for the Ansible Tower install|yes||
+|ansible_tower.workflow_templates.name|Name of the Job Template|yes||
+|ansible_tower.workflow_templates.description|Description of the Workflow Template	|no||
+|ansible_tower.workflow_templates.permissions|Permissions to run the workflow (see below)	|no||
+|ansible_tower.workflow_templates.allow_simultaneous|Allows multiple instances of the workflow to run in parallel|no|true|
+|ansible_tower.workflow_templates.nodes|Structure to define the job execution of the workflow (see below)	|no||
+
+**_Note:_** Workflow Template configuration will **only** happen if the `ansible_tower.workflow_templates` portion of the dictionary is defined. Likewise, the installation expects this section to be "complete" if specified as it otherwise may error out.
+
+### Workflow Nodes
+The Workflow Template nodes structure is defined in the `workflow_templates.nodes` variable. This consists of several variables which should be defined in a tree structure. The variables for each node is explained below.
+
+| Variable | Description | Required | Defaults |
+|:---------|:------------|:---------|:---------|
+|unified_job_template.name|Name of the job_template to use for the node| yes| no ||
+|success_nodes|List of job nodes to be run after successful execution| no| no ||
+|no_nodes|List of job nodes to be run after failure execution| no| no ||
+
+An example of such an inventory is shown below.
+
+```yaml
+ansible_tower:
+  workflow_templates:
+  - name: Example Workflow Template
+    nodes:
+      - unified_job_template:
+          name: "Foo"
+        success_nodes:
```

Comment:
Probably would want to include at least one example of an extra level of nodes so it's understood that it can have multiple levels, not just one as shown in all of these samples.